docs/install/macos.md

```diff
@@ -28,26 +30,30 @@ all users.
    It is recommended that you re-login, so that your environment knows about `elan`.
    (Alternatively, type `source $HOME/.elan/env` to update the current terminal.)
 
-### Installing mathlib supporting tools
 
-In order to use mathlib supporting tools, you need to [get python](https://www.python.org/downloads/) first.
-Then, at a terminal, run the command
+Installing mathlib supporting tools
+---
+
+At a terminal, run the command
   ```bash
   curl https://raw.githubusercontent.com/leanprover-community/mathlib-tools/master/scripts/remote-install-update-mathlib.sh -sSf | bash
   ```
+(This will install tools that, amongst other things, let you download compiled binaries for mathlib.)
 
-### Installing and configuring an editor
+Installing and configuring an editor
+---
 
-There are two Lean-aware editors, VS Code and emacs.
+There are two editors you can use with Lean, VS Code and emacs.
 This document describes using VS Code (for emacs, look at https://github.com/leanprover/lean-mode).
 
 1. Install [VS Code](https://code.visualstudio.com/).
 2. Launch VS Code.
-3. Click on the extension icon in the view bar at the left
+3. Click on the extension icon ![(image of icon)](extensions-icon.png) in the side bar on the left edge of the screen
+   (or press <kbd>⇧ Shift</kbd><kbd>⌘ Command</kbd><kbd>X</kbd>)
```

Comment:
Could you add that icon picture in other installation files that mention it?

Reply:
Done. (I didn't do it for Debian, as we give command line instructions on that page.)